src/js/components/Chart/Chart.js

```diff
@@ -269,7 +269,12 @@ const Chart = React.forwardRef(
               }
               opacity={
                 (valueOpacity && theme.global.opacity[valueOpacity]) ||
-                valueOpacity
+                // eslint-disable-next-line no-nested-ternary
+                (valueOpacity === true
+                  ? theme.global.opacity.medium
+                  : valueOpacity === false
+                  ? undefined
+                  : valueOpacity)
```

Comment:
Since this code is being duplicated, I'm thinking we should extract it to a function.

Reply:
Fixed in latest push